spec/rails_helper.rb

```diff
@@ -1,8 +1,9 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
 
-rails_version = ENV.fetch('APPRAISAL_RAILS_VERSION', '4.2.10')
-require File.expand_path("../../spec/test_apps/rails-#{rails_version}/config/environment", __FILE__)
+require 'rails'
+require 'rails-controller-testing'
+require_relative "test_apps/rails-#{Rails.gem_version.segments.take(2).join('.')}/config/environment"
```

Comment:
Instead of setting and reading an environment variable for the Rails version, just obtain it from the `Rails.gem_version` method.

Here we're taking just the first two parts. eg Rails `6.1`.